src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,63 @@
+# Common data types
+
+## Preprocessed or cleaned data
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        <datatype>/
+            <source_keywords>[_space-<space>][_desc-<label>]_<suffix>.<ext>
+```
+
+Preprocessing in this context means transformations of data that do not change
+the number of dimensions of the input and are not explicitly covered by other
+data types in the specification.
```

Comment:
I don't love the _that do not change the number of dimensions_.

For instance, a DWI dataset with, e.g., two full samplings in opposed PE directions (i.e. 5D), will result in one 4D dataset that will be designated THE derivative of that DWI, while this is not currently observed in the DWI derivatives spec. This definition will force us to be super-cautious for every single derivative and there is large potential for oversights.

I think here we want to allude to whether a derivative file may keep the `suffix` of the original data or consider that the data type (as expressed by the suffix) has changed.

```suggestion
Preprocessing in this context means transformations of data that do not change
the data type of the input (as expressed by its BIDS `suffix`).
For instance, a change in the number of dimensions is likely to disrupt the propagation
of the input's `suffix` and thus, cannot be considered _preprocessed_ or _cleaned data_.